packages/@okta/vuepress-site/use_cases/inline_hooks/saml_hook/saml_hook/index.md

```diff
@@ -0,0 +1,391 @@
+
+| Property | Description                                                              | Data Type       |
+|----------|--------------------------------------------------------------------------|-----------------|
+| type     | One of the [supported commands](#supported-commands).                    | String          |
+| value    | Operand to pass to the command. It specifies a particular op to perform. | [value](#value) |
+
+#### Supported Commands
+
+The following command is currently supported for the SAML Assertion Inline Hook type:
+
+| Command                  | Description              |
+|--------------------------|--------------------------|
+| com.okta.assertion.patch | Modify a SAML assertion. |
+
+#### value
+
+The `value` object is where you specify the specific operation to perform. It is an array, allowing you to request more than one operation.
+
+| Property | Description                                                                                                                                                                                                           | Data Type       |
+|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
+| op       | The name of one of the [supported ops](#list-of-supported-ops).                                                                                                                                                       | String          |
+| path     | Location, within the assertion, to apply the operation. See [Specifying Location within the Assertion](##specifying-location-within-the-assertion) below. | String          |
+| value    | Value to set the claim to.                                                                                                                                                                                            | Any JSON object |
+
+#### List of Supported Ops
+
+| Op      | Description                             |
+|---------|-----------------------------------------|
+| add     | Add a new claim to the assertion.       |
+| replace | Modify an existing attribute statement. |
+
+### Specifying Location within the Assertion
+
+You specify the location within the assertion at which to apply your operation using a slash-delimited path.
+
+When performing an `add` op to add a new attribute statement, this will always begin with `/claims/` and be followed by the name of the new attribute you are adding.
+
+When modifying an existing assertions statement, the path could begin with `/subject/`, `/authentication/`, `/conditions/`, or `/claims/`, depending on which part of the assertion you want to modify. You then drill down within the child elements using slash-delimited element names, e.g., `/claims/array/attributeValues/1/value`.
```

Comment:
You should specify that:
1. This is following JSON Patch conventions
2. Explain that the `/1/` is specifying the index in an array